.github/workflows/build.yaml

```
@@ -174,11 +179,15 @@ jobs:
       - name: push
         run: |
           cd charts
+          short_ver=sha-${GITHUB_SHA::7}
```

There is some chance the short sha is longer than 7 characters. Something like 1/2^21 for any given sha? It is a function of the number of commits though so odds will increase over time.

```suggestion
          short_ver=sha-$(git rev-parse --short)
```

Another option is to refactor this so it somehow gets the output of the appropriate metadata actions, e.g. by having this run as part of the relevant image publish jobs.

---

@dmihalcik-virtru Yeah since GH gives you the full SHA (but sadly not the short SHA) as an env I was trying to be cute and use that and not recalc/require in-container Git binary.

https://stackoverflow.com/a/18134919 indicates that 8 should be way more than plenty for a repo of this size.